### DISCUSSION TITLE
New filament blade component `Alert`

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#10265] [Ideas]
Soon I will add a **PR**

**Final Result:**

<img width="949" alt="Screenshot 2023-12-12 at 18 24 10" src="https://github.com/filamentphp/filament/assets/121255432/0832b158-f5a4-4d0e-9660-9e5d39386630">

<img width="949" alt="Screenshot 2023-12-12 at 18 24 22" src="https://github.com/filamentphp/filament/assets/121255432/3ba7664d-beb8-48bb-8fe7-f14e48c69702">

**With actions:**

<img width="939" alt="Screenshot 2023-12-12 at 18 42 53" src="https://github.com/filamentphp/filament/assets/121255432/eb6c0803-9a81-40df-b3a6-db057508ee1f">

**Comment by sh4msi** [comment]
hi,

When do you want to do this?
I am waiting for it

@devhoussam1998

**Reply by devhoussam1998** [reply]
coming soon my friend I will make a PR the first january

**Reply by octaviogamero27** [reply]
We're waiting for it 
When do you launch it?

**Reply by JamieCee20** [reply]
Any update?
